src/server/pfs/server/pfsserver.proto

```diff
@@ -53,7 +53,11 @@ message PutFileURLTask {
   string dst = 1;
   string datum = 2;
   string URL = 3;
-  repeated string paths = 4;
+  reserved 4;
```

Comment:
This isn't needed, task service protos can be changed arbitrarily across versions.